### DISCUSSION TITLE
Time balance on prepaid hour packages

### DISCUSSION THREAD

**Discussion by olekrisek** [kimai/kimai#4278] [General] [answered]
I am selling pre-paid packages of 20, 40 or 60 hours, connected to a customer. As project progresses, a balance report showing how much is left on the customer package. Some activities, like meetings are to be calculated as 60% time, so only 60% of the actual time has to be deducted from the customers package. 
Is there any good solution for this using Kimai ? - Now I am exporting it to a excel spreadsheet and import into a FilMaker database, and do the time-calculations and reporting from there.

**Comment by kevinpapst** [comment, marked as answer]
Convert time into money.

Create a new project for every package someone buys, e.g. 20 hours = 2000 $ budget on that project.

Regular activities = 100 $ per hour
Meetings = 60 $ per hour

Kimai does the rest for you.

**Reply by olekrisek** [reply]
Thank you. That was an good idea :-)
